lib/data/network/dio_wrapper.dart

```diff
@@ -4,8 +4,8 @@ part of auth0;
 class DioWrapper {
   final Dio dio = Dio();
   final JsonDecoder _decoder = JsonDecoder();
-  String host = "";
-  String scheme = "";
+  late String host;
+  late String scheme;
 
```

Comment:
Added the late keyword instead of initializing it as an empty string since they are initialized soon after the constructor call.